recipes/ruy/all/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 2.8.11)
```

CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS needs cmake >=3.4
```suggestion
cmake_minimum_required(VERSION 3.4)
```